lisa/sut_orchestrator/azure/features.py

```diff
@@ -596,6 +600,103 @@ def get_raw_data_disks(self) -> List[str]:
             disk_array[int(disk.split("/")[-1].replace("lun", ""))] = cmd_result.stdout
         return disk_array
 
+    def add_managed_data_disk(
+        self,
+        names: List[str],
+        luns: List[str],
+        size: str = "20",
+        location: str = "westus2",
```

Comment:
The location can be gotten from node context, not pass it in as a parameter.

Reply:
Awesome. Thanks!